integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java

```diff
@@ -203,10 +203,14 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     createJobToChangePermissionsOnPvHostPath(pvName, pvcName, domainNamespace);
 
     // create the domain CR with a pre-defined configmap
+    // setting setDataHome to false, testMiiRemoveTarget fails when data home is set at domain resource level
+    // because of bug https://jira.oraclecorp.com/jira/browse/OWLS-88679
```

Comment:
Please no internal links in the codebase :)

Reply:
Just to clarify, I'm okay if you reference the JIRA (so that we can find it), but just without the link.